pkg/jobs/adopt.go

```diff
@@ -511,7 +511,7 @@ const pauseAndCancelUpdate = `
 					last_run = NULL
     WHERE (status IN ('` + string(StatusPauseRequested) + `', '` + string(StatusCancelRequested) + `'))
       AND ((claim_session_id = $1) AND (claim_instance_id = $2))
-RETURNING id, status
+RETURNING id, status, job_type
```

Comment:
i don't think we should change this query for this pr-- we can grab the job type from the in memory job object

Reply:
@dt are you ok changing this query for this PR?